guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala

```diff
@@ -47,26 +46,72 @@ class I18nDemosTest extends SeleniumTest {
 
     "contain working frontend translations demo" in {
       driver.navigate().refresh()
-      runDemo("frontend-translations-demo")()
+
+      //runDemo("frontend-translations-demo")()
+      def demo = driver.findElementById("frontend-translations-demo")
+
+      eventually {
+        val elements = demo.findElements(new ByCssSelector("li"))
+        val translations: Map[String, String] = elements.asScala.map(item => {
+          val parts = item.getText.split(":")
+          (parts(0).trim, parts(1).trim)
+        }).toMap[String, String]
+
+        verifyEnTranslations(translations)
+      }
     }
 
-    //todo fix after new, Atmosphere-less RPC is merged
-    "contain working remote translations demo" ignore {
+    "contain working remote translations demo" in {
       driver.navigate().refresh()
-      val translations = runDemo("rpc-translations-demo")()
-      verifyPlTranslations(translations)
+
+      //val translations = runDemo("rpc-translations-demo")()
+      //verifyPlTranslations(translations)
+      def demo = driver.findElementById("rpc-translations-demo")
+
+      eventually {
+        val elements = demo.findElements(new ByCssSelector("li"))
+        val translations: Map[String, String] = elements.asScala.map(item => {
```

Comment:
this translations can be extracted to a method - it's used in 3 places

Reply:
IDC as it's going to be removed anyway. The new version of this test extracts it.